Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN rm -f /tmp/trivy.deb
 
 # Install OSV-Scanner from Datadog
 RUN mkdir /osv-scanner
-RUN curl -L -o /osv-scanner/osv-scanner.zip https://github.com/DataDog/osv-scanner/releases/download/v0.1.0/osv-scanner_0.1.0_linux_amd64.zip >/dev/null 2>&1 || exit 1
+RUN curl -L -o /osv-scanner/osv-scanner.zip https://github.com/DataDog/osv-scanner/releases/download/v0.4.1/osv-scanner_linux_amd64.zip >/dev/null 2>&1 || exit 1
```

Comment:
Can we pull the latest release instead?

Reply:
No. If there is a breaking change published in `osv-scanner`, it will break it. We should instead pin the version.